website/content/en/docs/building-operators/golang/references/logging.md

```diff
@@ -33,7 +33,7 @@ Operators set the logger for all operator logging in [`cmd/manager/main.go`][cod
 package main
 
 import (
-	"sigs.k8s.io/controller-runtime/pkg/log/zap"  
+	"sigs.k8s.io/controller-runtime/internal/log/zap"  
```

Comment:
This is a controller-runtime dependency, ditto below

Reply:
I got overzealous with my replacements. Fixed.